low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -0,0 +1,159 @@
+        if not os.path.exists(file_store_config_path):
+            shutil.copyfile(f"{SSPL_BASE_DIR}/conf/sspl.conf.{PRODUCT}", file_store_config_path)
+
+        # For Dev environment
+        if ENVIRONMENT == "DEV":
+            shutil.copyfile(f"{SSPL_BASE_DIR}/conf/sspl.conf.{PRODUCT}", file_store_config_path)
+
+        # Copy rsyslog configuration
+        if not os.path.exists(self.RSYSLOG_CONF):
+            shutil.copyfile(f"{SSPL_BASE_DIR}/low-level/files/{self.RSYSLOG_CONF}", self.RSYSLOG_CONF)
+
+        if not os.path.exists(self.RSYSLOG_SSPL_CONF):
+            shutil.copyfile(f"{SSPL_BASE_DIR}/low-level/files/{self.RSYSLOG_SSPL_CONF}", self.RSYSLOG_SSPL_CONF)
+
+        # Create soft link for SINGLE product name service to existing LDR_R1, LDR_R2 service
+        # Instead of keeping separate service file for SINGLE product with same content.
+        currentProduct = f"{SSPL_BASE_DIR}/conf/sspl-ll.service.{PRODUCT}"
+        if (PRODUCT == "SINGLE" and not os.path.exists(currentProduct)) or (PRODUCT == "DUAL" and not os.path.exists(currentProduct)):
+            os.symlink(f"{SSPL_BASE_DIR}/conf/sspl-ll.service.{PRODUCT_NAME}", currentProduct)
+
+        if PRODUCT == "CLUSTER" and not os.path.exists(currentProduct):
+            os.symlink(f"{SSPL_BASE_DIR}/conf/sspl-ll.service.LDR_R2", currentProduct)
+
+        # Copy sspl-ll.service file and enable service
+        shutil.copyfile(currentProduct, "/etc/systemd/system/sspl-ll.service")
+        Service('dbus').process('enable', 'sspl-ll.service')
+        daemon_reload_cmd = "systemctl daemon-reload"
+        output, error, returncode = SimpleProcess(daemon_reload_cmd).run()
+        if returncode != 0:
+            raise PostInstallError(returncode, error, daemon_reload_cmd)
+
+        # Copy IEC mapping files
+        os.makedirs(f"{PRODUCT_BASE_DIR}/iem/iec_mapping", exist_ok=True)
+        distutils.dir_util.copy_tree(f"{SSPL_BASE_DIR}/low-level/files/iec_mapping/",
+            f"{PRODUCT_BASE_DIR}/iem/iec_mapping")
+
+        # Skip this step if sspl is being configured for node replacement scenario as sspl configurations are
+        # already available in consul on the healthy node
+        # Running script for fetching the config using salt and feeding values to consul
+        # Onward LDR_R2, consul will be abstracted out and it won't exit as hard dependeny of SSPL
+        if PRODUCT_NAME == "LDR_R1":
+            if not os.path.exists(REPLACEMENT_NODE_ENV_VAR_FILE):
+                config_from_salt = f"{self._script_dir}/sspl_fetch_config_from_salt.py {ENVIRONMENT} {PRODUCT_NAME}"
+                output, error, returncode = SimpleProcess(config_from_salt).run()
+                if returncode != 0:
+                    raise PostInstallError(returncode, error, config_from_salt)
+        else:
+            load_sspl_config = f"{self._script_dir}/load_sspl_config.py -C {file_store_config_path}"
```

Comment:
Avoid introducing any string formatting using f" in new code, it would be creating problem during localization efforts later.

Reply:
Done.